share/man/man9/intro.9

```diff
@@ -410,9 +410,9 @@ and
 The interfaces for declaring loadable kernel modules are described by
 .Xr module 9 .
 .Ss Interrupts
-The machine-independent portion of the interrupt framework supporting the
-registration and execution of interrupt handlers is described by
-.Xr intr_event 9 .
```

Comment:
LanguageTool drew attention to passive voice in a few areas.

Ignoring LanguageTool, this one sentence caught my eye.

Reply:
Thanks. I am prone to this weak type of phrasing.